### ISSUE TITLE
About data

### ISSUE THREAD

**Issue by guolinzhou1998** [Uzoma-N/Transformer-Oil-Analysis#1]
Hello, I am very interested in your project, can you give a brief introduction of the data used in the project

**Comment by Uzoma-N** [comment]
The data contains sensor readings. The gaseous composition of the transformer oil is taken at specified time stamps. You can ask a more specific question.

**Comment by guolinzhou1998** [comment]
> The data contains sensor readings. The gaseous composition of the transformer oil is taken at specified time stamps. You can ask a more specific question.

Thank you for your reply! Does each table in this project correspond to a different transformer? Also some of the gas data in these tables seem to be incomplete.

**Comment by Uzoma-N** [comment]
Yes, each table represents transformer oil data gotten from different transformers. The data is raw and that explains why it is incomplete.
